ipaserver/plugins/service.py

```diff
@@ -604,6 +604,11 @@ class service_add(LDAPCreate):
             label=_('Force'),
             doc=_('force principal name even if not in DNS'),
         ),
+        Flag('avoid_host_check',
```

Comment:
For naming consistency: there is no precedence of `avoid_xxx` option. But we have `skip_overlap_check` and `skip_nameserver_check` in `dnszone-add` and  and `ignore_last_of_role` and `ignore_topology_disconnect` (sever-del command). Given that there is `check` in the name using `skip_host_check` is more consistent